src/Functions/Sticky.js

```diff
@@ -0,0 +1,11 @@
+//  function for header animation
```

Comment:
I am just curious, if we need to separate all functions? In this case it was a little bit difficult to understand from where this new class 'sticky' comes. And also I would use ref instead of querySelector